entries/p/patient.labcorp.com.json

```
@@ -1,14 +1,21 @@
 {
   "Labcorp Patient": {
     "domain": "patient.labcorp.com",
-    "contact": {
-      "form": "https://www.labcorp.com/contact-us/labcorp-patient-portal"
-    },
+    "tfa": [
+      "sms",
+      "totp",
+      "custom-software"
+    ]
+    "custom-software": [
+      "Okta Verify"
+    ],
+    "documentation: "https://patient.labcorp.com/help-center/q-and-a/8000/2"
```

Thanks for the pull request @andbenn!

Just a small correction to fix the syntax errors :wink: 
```suggestion
    ],
    "custom-software": [
      "Okta Verify"
    ],
    "documentation: "https://patient.labcorp.com/help-center/q-and-a/8000/2",
```